readme.md

```diff
@@ -13,7 +13,7 @@ Activate environment:
 ```
 source env/bin/activate
 ```
-Run several N - 1 clients by:
+Run N - 1 clients by:
```

Comment:
Thank you. It must be "Run first N-1 clients by:"

Reply:
Do you want to fix your pull request or do I need to fix it myself in `master`?

Reply:
Fixed, but choose "squash and merge" when will be merging

Reply:
Also as I can you can edit pull request by yourself